### DISCUSSION TITLE
Chip.Group to allow deselect

### DISCUSSION THREAD

**Discussion by ItaiAxelrad** [mantinedev/mantine#4531] [Feature requests]
Currently, a chip group with `multiple={false}` does not allow deselect. Seeing as the alternatives like segmented control and radio group likely shouldn't allow deselect, chip group seems like a good contender for this functionality. This could be added using an `allowDeselect` prop in order to maintain existing behaviour. Thoughts?

**Comment by rtivital** [comment]
Chip component is built with `input[type="radio"]` which does not support deselection. If you need this feature, use controlled component with `multiple` prop and limit the number of selected items to 1.

**Reply by ItaiAxelrad** [reply]
Thanks for the quick reply! Ah, I see. I assumed it was a checkbox either way but `type: ctx.multiple ? 'checkbox' : 'radio'` makes more sense
